tests/functional/u/undefined/undefined_loop_variable.py

```diff
@@ -125,25 +130,33 @@ def for_else_continue(iterable):
         print(thing)
 
 
+def for_else_no_return(iterable):
+    def fail() -> NoReturn:
+        ...
+
+    while True:
+        for thing in iterable:
+            break
+        else:
+            fail()
+        print(thing)
+
+
 lst = []
 lst2 = [1, 2, 3]
 
 for item in lst:
     pass
 
-bigger = [
-    [
-        x for x in lst2 if x > item
-    ]
-    for item in lst
-]
+bigger = [[x for x in lst2 if x > item] for item in lst]
```

Comment:
Do you mind reverting this change? The variable checker depends on line numbers, so changing the line spacing can change the result.

Reply:
Good catch! I'll revert!